src/onboarding/Start.js

```diff
@@ -167,10 +180,11 @@ class StartContent extends React.PureComponent {
               <DropDown
                 items={networkChooserItems.map(([id, label]) => label)}
                 onChange={this.handleNetworkChange}
+                wide={responsiveLanding}
               />
             </div>
 
-            {network.type === 'main' && (
+            {!responsiveLanding && network.type === 'main' && (
```

Comment:
@jounih let's make sure to think about how to include this disclosure on smaller screens later.